src/main/scala/nl.knaw.dans.easy.deposit/authentication/TokenSupport.scala

```diff
@@ -56,7 +59,7 @@ trait TokenSupport extends DebugEnhancedLogging {
   def decodeJWT(token: String): Try[AuthUser] = {
     for {
       decoded <- Jwt.decode(token, tokenConfig.secretKey, Seq(tokenConfig.algorithm))
-      parsed <- fromJson(decoded)
+      parsed <- fromJson(decoded.toJson)
```

Comment:
```suggestion
      parsed <- fromJson(decoded.content)
```
change also 
https://github.com/DANS-KNAW/easy-deposit-api/blob/1713f181141efdbb526554175970f1ad871f83a2/src/main/scala/nl.knaw.dans.easy.deposit/authentication/TokenSupport.scala#L74
into
```
That should fix `SessionSpec` 
private case class Token(uid: String)
```